docs/groovy-script/getting_started/editors.md

```diff
@@ -42,6 +44,28 @@ This extension adds syntax validation, auto-completion for all installed mods, a
 6. Done
 
 
+## IntelliJ IDEA
+
+[IntelliJ IDEA](https://www.jetbrains.com/idea/) is an Integrated Development Environment created and distributed by JetBrains.
+The plugin will work with either Community Edition or Ultimate.
+
+[`Integer Limit`](https://github.com/IntegerLimit) has created a plugin, called [`GroovyScript`](https://plugins.jetbrains.com/plugin/25915-groovyscript),
```

Comment:
the reason why it is worded the way it is is because there are 3 types here - "author", "project", and "vendor" - and while in this case, the author and vendor are the same, they can be different. ill extract the author to a separate section thats more emphasized.